modules/kubeconfig/output.tf

```diff
@@ -1,6 +1,7 @@
 # Copyright © 2021-2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# tflint-ignore: terraform_standard_module_structure
```

Comment:
Can you clarify this exception?

Reply:
After reviewing the warning I believe that this is a false positive from tflint.

tflint is reporting this rule is being broken: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.2.2/docs/rules/terraform_standard_module_structure.md

> Terraform's documentation outlines a [Standard Module Structure](https://www.terraform.io/docs/modules/structure.html). A minimal module should have a main.tf, variables.tf, and outputs.tf file. Variable and output blocks should be included in the corresponding file.
---
The [kubeconfig](https://github.com/sassoftware/viya4-iac-aws/tree/main/modules/kubeconfig) module in our viya4-iac-aws project does have those three components so it didn't make sense as to why the warning was being thrown.

Reply:
You may just need to rename output.tf -> outputs.tf

Reply:
@dhoucgitter, good catch, I didn't even notice that. 
I renamed the file and reran the linter locally and the warning went away. I'll do another quick smoke test and push the changes. I'll let you guys know when this is ready for review again (tomorrow morning).